R/gdp_ids_and_units.R

```diff
@@ -0,0 +1,67 @@
+#'@title Get Geo Data Portal station ids and units 
+#'
+#'@description
+#'Parses the first three rows of a Geo Data Portal output file and returns a named list ready for use in creation of netCDF DSG files.
+#'
+#'@param gdp_file A Geo Data Portal output file containing one statistic, currently no support for multiple statistic output files.
+#'
+#'@return A named list as follows:\cr
+#'nc_stations contains unique station ids,\cr
+#'nc_data_var_unique_name contains the variables in the file,\cr
+#'unique_units contains the units associated with each variable.
+#'
+#'@export
+gdp_ids_and_units<-function(GDPFile)
```

Comment:
Yeah... I'm working through that now. another commit coming that will demonstrate what I've learned.